.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -0,0 +1,72 @@
+name: Wheels
+on:
+  push:
+    branches:
+      - master
+
+env:
+  CIBW_BUILD: 'cp37-* cp38-* cp39-*'
+
+  # MacOS specific build settings
+  CIBW_BEFORE_ALL_MACOS: |
+    brew uninstall --force oclint
+    brew install gcc libomp
+
+  # Python build settings
+  CIBW_BEFORE_BUILD: |
+    pip install pybind11
+
+  # Testing of built wheels
+  CIBW_TEST_REQUIRES: numpy scipy pytest pytest-cov pytest-mock flaky
+
+  # Note: breaking up tests so that no test cases hang on Windows, to be reverted
+  CIBW_TEST_COMMAND: |
+    pip install numpy==1.19.5 scipy
+    pl-device-test --device=lightning.qubit --skip-ops -x --tb=short --no-flaky-report
+
+  # Skip PPC tests due to lack of numpy/scipy wheel support
+  CIBW_TEST_SKIP: "*-manylinux_{aarch64}"
+
+  # Use Centos 7 wheel-builder for ARM, Debian for PPC
+  CIBW_MANYLINUX_AARCH64_IMAGE: manylinux2014
+
+jobs:
+  linux-wheels-aarch64:
+    if: github.ref == 'refs/heads/master'
```

Comment:
Is this a double check that we're only building on `master`?

Reply:
Indeed. These can be safely removed, though I'd rather keep the artifact upload checks as an additional safety measure.